tests/unit/cli/test_job_config_schema_as_args_parser.py

```diff
@@ -6,7 +6,7 @@
 class TestJobConfigSchemaAsArgsParser(unittest.TestCase):
     maxDiff = None
 
-    def setUp(self) -> None:
+    def setUp(self):
```

Comment:
Turns out this doesn't typecheck because odictliteral isn't mypy friendly.  Not sure why I had a `-> None` thre.